### DISCUSSION TITLE
My Wallet Does Not Synch at all (Never)

### DISCUSSION THREAD

**Discussion by kadir1305** [Chia-Network/chia-blockchain#3919] [Support]
Hello. My wallet does not synch. i tried everyting. firewall exceptions, reinstalling, deleting all chia related files and clean reinstall  but issue is still there and i cant farm even though i have 1 plot.  Synching doesnt even start at all. What should i do? ver 1.1.4

![ccc](https://user-images.githubusercontent.com/83759098/117278145-2a55d780-ae69-11eb-95ec-549fd89adc65.JPG)

**Comment by ghost** [comment]
Try to set log level to DEBUG, restart Chia and look into log.
- https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
- https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
